src/main/java/nextstep/subway/exception/ExceptionResponse.java

```diff
@@ -0,0 +1,20 @@
+package nextstep.subway.exception;
+
+public class ExceptionResponse {
+
+	private int status;
+	private String message;
```

Comment:
값을 재할당 해줄 필요가 없다면
final 을 사용해보면 어떨까요? 😄 
```suggestion
	private final int status;
	private final String message;
```